cmake/adjust/template/vendor/CXX_Clang.cmake

```diff
@@ -0,0 +1,51 @@
+include_guard(GLOBAL)
+
+# Debug
+set(NBL_CXX_DEBUG_COMPILE_OPTIONS
+	-ggdb3 -Wall -fno-omit-frame-pointer -fstack-protector-strong
+)
+
+# Release
+set(NBL_CXX_RELEASE_COMPILE_OPTIONS
+	-fexpensive-optimizations
+)
+
+# RelWithDebInfo
+set(NBL_CXX_RELWITHDEBINFO_COMPILE_OPTIONS "")
+
+# Global
+list(APPEND NBL_CXX_COMPILE_OPTIONS
+	-Wextra
+	-fno-strict-aliasing
+	-msse4.2
+	-maes
+	-mfpmath=sse		
+	-Wextra
+	-Wno-sequence-point
+	-Wno-unused-parameter
+	-Wno-unused-but-set-parameter
+	-Wno-c++98-compat
+	-Wno-c++98-compat-pedantic
+	-Wno-padded
+	-Wno-unsafe-buffer-usage
+	-Wno-switch-enum
+	-Wno-error=ignored-attributes
+	-Wno-error=unused-function
+	-Wno-error=unused-variable
+	-Wno-error=unused-parameter
+	-Wno-error=ignored-attributes
+	-Wno-error=non-pod-varargs
```

Comment:
I take it that we'll remove this over time?

Reply:
yes, @YasInvolved is going to kill all warnings